ebos/eclgenericcpgridvanguard.hh

```diff
@@ -170,6 +171,7 @@ protected:
     void allocCartMapper();
 
     void doCreateGrids_(EclipseState& eclState);
+    void createCpGridWithLgrs(const LgrCollection& lgrCollection, const int lgrsSize);
```

Comment:
Nitpick: The CPGrid is already created and we add the LGRs. Maybe change the name, e.g. addLgrsToGrid?